build/configs/artik053/README.md

```diff
@@ -50,6 +55,15 @@ openocd -f artik053.cfg -c ' \
     flash_write os ../build/output/bin/tinyara_head.bin; exit'
 ```
 
+### Using download script
+
+```bash
+make download ALL
+```
+
+This is same as programming the complete set of binaries at Using OpenOCD.
+
+
```

Comment:
I suggest to move the sequence,

explain make download ALL. and then describe details.
developer usually try to follow step by step in sqeuce, and it'll better to move needless operations as a kind of appendix area or the bottom of the description